code/modules/food_and_drinks/drinks/drinks.dm

```diff
@@ -13,6 +13,7 @@
 	possible_transfer_amounts = list(5,10,15,20,25,30,50)
 	volume = 50
 	resistance_flags = 0
+	you_drink_from_this = TRUE
```

Comment:
Why are you setting this here??

Reply:
For some reason it only worked when I did this